source/help/mirrors.md

```diff
@@ -1,14 +1,8 @@
 arXiv mirror sites
 ==================
 
-In addition to the [main site](https://arxiv.org), there are mirror sites for arXiv content. These are
-updated daily but have fewer features than the main site:
+The mirror network has been shut down as of September 15th, 2024.
 
--   [cn.arXiv.org](http://cn.arxiv.org/) (China)
--   [de.arXiv.org](http://de.arxiv.org/) (Germany)
--   [in.arXiv.org](http://in.arxiv.org/) (India)
--   [lanl.arXiv.org](http://lanl.arxiv.org/) (née xxx.lanl.gov, U.S. mirror at Los Alamos)
+The arXiv mirrors were begun as an attempt to provide better local access to the arXiv corpus to facilitate speed during the nascent modern internet. The mirror network was significantly expanded in the late 1990's and, at the time, was heavily used. Geographic locality is now much less useful than it was when the mirror network was established. Now, most users geographically close to mirrors, perhaps in neighboring countries, have better access to the main site than to a "nearby" mirror. These read-only mirrors took significant effort of both arXiv staff and hosting volunteer institutions to maintain, implement and update. 
```

Comment:
> nascent modern internet

"nascent modern" ? I'm not quite sure if I've ever seen those two words used together before.

Am I a nascent modern man?

Reply:
I'm an innovator when it comes to misusing text. How about: 

> ...in arXiv's early life. 

instead?